src/integrationTest/java/uk/gov/hmcts/darts/event/controller/EventSearchControllerTest.java

```diff
@@ -94,12 +94,22 @@ void allowsSuperUsers() throws Exception {
 
     @Test
     void returnsAllFieldsCorrectly() throws Exception {
+        int eventHearingsCount = 2;
+        int eventsCount = 2;
         given.anAuthenticatedUserWithGlobalAccessAndRole(SUPER_ADMIN);
-        List<EventEntity> entity = eventsGivensBuilder.persistedEvents(1);
+        List<EventEntity> entity = eventsGivensBuilder.persistedEventsWithHearings(eventsCount, eventHearingsCount);
 
-        CourtCaseEntity courtCaseEntity = entity.get(0).getHearingEntities().get(0).getCourtCase();
-        courtCaseEntity.setDataAnonymisedTs(now());
-        dartsDatabase.save(courtCaseEntity);
+        // assign a case anonymised date
+        for (int i = 0; i < eventsCount; i++) {
+            for (int j = 0; j < eventHearingsCount; j++) {
+                CourtCaseEntity courtCaseEntity = entity.get(i).getHearingEntities().get(j).getCourtCase();
+                courtCaseEntity.setDataAnonymised(true);
+                courtCaseEntity.setDataAnonymisedTs(now());
+
+                // associated another hearing to the event
+                dartsDatabase.save(courtCaseEntity);
+            }
+        }
```

Comment:
this is no longer needed now with the updated requirements

Reply:
This does affect the assertion as the date anonymised is now null. I have removed the above as well as the assertion as a result.